src/util.js

```diff
@@ -162,7 +172,14 @@ export function memoize<F : Function>(method : F, options? : MemoizeOptions = ge
             cache = simpleCache = simpleCache || {};
         }
 
-        const cacheKey = serializeArgs(args);
+        let cacheKey;
+
+        try {
```

Comment:
@bluepnume So if can't serialize args then just execute the method?

Reply:
Yup, that was an excellent idea from @gregjopa. It probably makes more sense to have the function work in a non-memoized way, than to completely fail at this point.

Reply:
Thanks 😄  Should we also use this same fallback approach in memoizePromise()? https://github.com/krakenjs/belter/blob/master/src/util.js#L211

Reply:
Probably, but let's take that as a separate PR to de-risk, since memoizePromise doesn't have any tests yet.